tests/conftest.py

```diff
@@ -0,0 +1,17 @@
+import os
+import pytest
+
+
+@pytest.fixture
+def set_envars(request):
+    """Set and unset one or more envrionment variables"""
```

Comment:
```suggestion
    """Set and unset one or more environment variables"""
```
I'm starting to replace @swadeley

Reply:
Also I hope that wasn't Copilot...

Reply:
haha nope. all me!